functional/vfio/run.sh

```diff
@@ -25,20 +25,34 @@ HYPERVISOR=
 MACHINE_TYPE=
 IMAGE_TYPE=
 
+# Option to choose an alternative PCI device for the VFIO test
+VFIO_PCI_CLASS=${VFIO_PCI_CLASS:-"Ethernet controller"}
+VFIO_PCI_NAME=${VFIO_PCI_NAME:-"Virtio.*network device"}
+VFIO_CHECK_GUEST_KERNEL=${VFIO_CHECK_GUEST_KERNEL:-"ip a | grep \"eth\" || die \"Missing VFIO network interface\""}
+VFIO_HOTPLUG=${VFIO_HOTPLUG:-"bridge-port"}
+VFIO_COLDPLUG=${VFIO_COLDPLUG:-"bridge-port"}
```

Comment:
Won't this result in us always configuring both `hot_plug_vfio` and `cold_plug_vfio`? What will be the result of that?

Reply:
The shim will fail saying you've set both which is not working, good catch.

Reply:
Added the correct defaults PTAL